### PR TITLE
added vim inline check support, documentation

### PR DESCRIPTION
**Pull request by samharad**

### CODE REVIEW

File: vim/cpp/lint381.vim [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  " 1) Ensure that you have Syntastic installed: https://github.com/vim-syntastic/syntastic
+  " 2) Place this file in ~/.vim/bundle/syntastic/syntax_checkers/cpp
+  " 3) Place the c/lint381.vim file in ~/.vim/bundle/syntastic/syntax_checkers/c
+  " 4) Add these lines to your vimrc: 
+    " let g:syntastic_cpp_checkers = ['lint381']
+    " let g:syntastic_c_checkers = ['lint381']
+    " Of course, precede 'lint381' with any other linters you may wish to use, e.g.:
+      " let g:syntastic_cpp_checkers = ['cppcheck', 'lint381']
+
+  " Note that if you have multiple checkers enabled, the defualt behavior of Syntastic
```
Comment (arxanas) [inline vim/cpp/lint381.vim]: defualt -> default

**Review by arxanas — APPROVED** [review]
I don't use Syntactic anymore but I'll assume from the screenshot that it works. Thanks for contributing :)

**Comment by arxanas** [comment]
Closes #5. We'll see if commenting that will automatically close that issue when this PR is merged.

**Comment by samharad** [comment]
Good catch. This is a cool project... I may have to take compilers if I survive 381!

**Review by arxanas — APPROVED** [review]

**Pull request merged**